app.plugin.js

```diff
@@ -0,0 +1,30 @@
+const { withGradleProperties, withAppBuildGradle } = require('@expo/config-plugins')
```

Comment:
Setup is ok, but if you want to set it up using TypeScript, you can look here: https://github.com/animo/expo-ausweis-sdk. Contains both an expo module and plugin

Reply:
Ah i see you already did this in the ios branch :)